src/odb/src/def/def/defiPath.cpp

```diff
@@ -602,7 +605,7 @@ void defiPath::print(FILE* fout) const
 void defiPath::bumpSize(int size)
 {
   int i;
-  int* newKeys = (int*) malloc(size * sizeof(int*));
+  int* newKeys = (int*) malloc(size * sizeof(int));
```

Comment:
@osamahammad21 would you help out with debugging this issue?  This change is correct but the result is that drc_test.tcl fails (adding back the '*' "fixes" the test).  It appears that test is sensitive to memory layout which it shouldn't be.  I'm not clear where it starts to diverge and would appreciate your insight.  It is quite consistent for me.

Reply:
so it looks like previously it allocated twice the size (on a 64 bit machine) ? So if a test that previously passed and now failed, there is probably some out-of-bounds writing, and it accidentally was not noticed because there was more memory at the end.

Maybe `asan` or `valgrind` should be the next steps then.

Reply:
I already tried valgrind but it found nothing.

Reply:
@luis201420 this is the case I mentioned of changing memory allocation affecting results